eosmetrics/emtr-aggregate-timer.c

```diff
@@ -106,21 +107,20 @@ on_server_aggregate_timer_started_cb (GObject      *source_object,
 
   if (error)
     {
-      g_warning ("Error creating aggregate timer: %s", error->message);
+      if (!g_error_matches (error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
+        g_warning ("Error creating aggregate timer: %s", error->message);
```

Comment:
I think with this change if you construct a timer and then immediately destroy it:

```c
g_object_unref (emtr_event_recorder_start_aggregate_timer (emtr_event_recorder_get_default ()))
```

then the daemon will receive the StartAggregateTimer call, dutifully return an object path, and then the timer will still be running until the client process terminates.